From 463c5f215b880480325fba4f43dde8e5d0e9ae14 Mon Sep 17 00:00:00 2001
From: Omri Itach <omrii@marvell.com>
Date: Thu, 3 Dec 2015 16:06:07 +0200
Subject: [PATCH 0571/1240] dts: a3700: add customer0 device tree

Added customer0 dts file, and proper Makefile entry

Board configuration:
SPI0, SGMII0 with PHY address = 0x1, SATA

Change-Id: Ice3296467c44f8f8495a8d38a4528d0d943dd5be
Signed-off-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/25595
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm/dts/Makefile                |  3 +-
 arch/arm/dts/armada-lp-customer0.dts | 57 ++++++++++++++++++++++++++++++++++++
 2 files changed, 59 insertions(+), 1 deletion(-)
 create mode 100644 arch/arm/dts/armada-lp-customer0.dts

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 0c6762b..32227ef 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -53,7 +53,8 @@ dtb-$(CONFIG_SOCFPGA) += socfpga_cyclone5_socrates.dtb
 
 dtb-$(CONFIG_TARGET_ARMADA_38X) += armada-385-db.dtb
 dtb-$(CONFIG_TARGET_ARMADA_LP) += armada-lp-db.dtb \
-				  armada-lp-palladium.dtb
+				  armada-lp-palladium.dtb \
+				  armada-lp-customer0.dtb
 dtb-$(CONFIG_TARGET_ARMADA_8K) += apn-806-db.dtb \
 		       apn-806-pd.dtb \
 		       apn-806-a0-pd.dtb \
diff --git a/arch/arm/dts/armada-lp-customer0.dts b/arch/arm/dts/armada-lp-customer0.dts
new file mode 100644
index 0000000..5762caf
--- /dev/null
+++ b/arch/arm/dts/armada-lp-customer0.dts
@@ -0,0 +1,57 @@
+
+/dts-v1/;
+
+#include "armada-lp.dtsi"
+
+/ {
+	model = "DB-88F37XX-Customer0";
+	compatible = "marvell,armada-lp-db0", "marvell,armada-lp";
+
+	soc {
+		internal-regs {
+			serial: serial@12000 {
+				status = "okay";
+			};
+			spi0: spi@10600 {
+				status = "okay";
+			};
+			neta0: neta@30000 {
+				status = "okay";
+				phy_addr = <0x1>;
+				phy_mode = "sgmii";
+			};
+			neta1: neta@40000 {
+				status = "disabled";
+				phy_addr = <0x0>;
+				phy_mode = "rgmii";
+			};
+			i2c: i2c@11000 {
+				status = "okay";
+			};
+			comphy {
+				phy0 {
+					phy-type = <PHY_TYPE_SGMII0>;
+					phy-speed = <PHY_SPEED_1_25G>;
+				};
+				phy1 {
+					phy-type = <PHY_TYPE_UNCONNECTED>;
+					phy-speed = <PHY_SPEED_1_25G>;
+				};
+			};
+			pcie-controller {
+				pcie@1,0 {
+					status = "disabled";
+				};
+			};
+			sata: sata@e0000 {
+				status = "okay";
+			};
+			mmc0: mmc@D8000 {
+				status = "disabled";
+			};
+			usb3: usb3@58000 {
+				status = "disabled";
+			};
+		};
+	};
+};
-- 
1.9.1

